doc/adr/2019-03-26-binary-distributions.md

```diff
@@ -0,0 +1,24 @@
+**Title**: We will not distribute/own/maintain binary packages, but we will
+support others doing so within reason.
+
+**Status**: in review
+
+**Context**: Compiling google-cloud-cpp from source is not always possible nor
+desired by all users. Some users may want to install a binary package that
+contains our public headers and pre-compiled libraries so that they can link
+against our library. There is no single standard binary package manager in C++,
+instead there are a variety of others that users might want to use (e.g., dpkg,
+rpm, vcpkg).
+
+**Decision**: We will not directly support nor provide any binary packages. We
+will not test binary distributions of our code. And we will not host configs
+for binary distributions, since that would involve hosting files which we do
+not test. However, we will do our best to make our code easily packageable by
+others in a wide variety of formats. For more context, see #333.
```

Comment:
nit: `#333` is not converted to a link automatically in `.md` files, use `[#333](https://github.com/googleapis/google-cloud-cpp/issues/333)`

Reply:
Fixed.